src/App.tsx

```diff
@@ -15,37 +15,83 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+const sortFieldAlphabet = 'alphabet';
+const sortFieldLength = 'length';
+const reverseFieldGoods = 'reverse';
+
+export const App = () => {
+  const [visibleGoods, setVisibleGoods] = useState(goodsFromServer);
+  const [activeSort, setActiveSort] = useState('');
+  const [originalGoods, setOriginalGoods] = useState(goodsFromServer);
+
+  const resetGoodsOfOrder = () => {
+    setVisibleGoods(goodsFromServer);
```

Comment:
Currently, `resetGoodsOfOrder` resets `visibleGoods` to `goodsFromServer`. Consider using `originalGoods` to reset `visibleGoods` to ensure that any future changes to the initial state are correctly handled. This will make the code more robust and maintainable.